app/mailers/subscription_mailer.rb

```diff
@@ -8,6 +8,7 @@ def digest(sub, dig)
     @user = sub.user
     @digest = dig
     @subscription = sub
+    @collections = Collection.left_outer_joins(:collaborators).where(collaborators: { id: @user.id}).or(Collection.where(user_id: @user.id)).distinct
```

Comment:
This might be nice to have as a method in `User`. `maintained_collections` or something?

Also it might be good to have a check here for `TeSS::Config.feature['collections']`, and return [] if not, just in case collections were previously enabled and then disabled after some collections were already created.